src/ServiceControl/LoggingConfigurator.cs

```diff
@@ -42,60 +36,32 @@ public static void ConfigureLogging(LoggingSettings loggingSettings)
-                ArchiveAboveSize = 30 * megaByte
-            };
-
             var consoleTarget = new ColoredConsoleTarget
             {
+                Name = "console",
                 Layout = simpleLayout,
+                DetectConsoleAvailable = true,
+                DetectOutputRedirected = true,
                 UseDefaultRowHighlightingRules = true
             };
 
-            var nullTarget = new NullTarget();
-
-            // There lines don't appear to be necessary.  The rules seem to work without implicitly adding the targets?!?
-            nlogConfig.AddTarget("console", consoleTarget);
-            nlogConfig.AddTarget("debugger", fileTarget);
-            nlogConfig.AddTarget("raven", ravenFileTarget);
-            nlogConfig.AddTarget("bitbucket", nullTarget);
-
             // Always want to see license logging regardless of default logging level
             nlogConfig.LoggingRules.Add(new LoggingRule("Particular.ServiceControl.Licensing.*", LogLevel.Info, fileTarget));
-            nlogConfig.LoggingRules.Add(new LoggingRule("Particular.ServiceControl.Licensing.*", LogLevel.Info, consoleTarget)
-            {
-                Final = true
-            });
+            nlogConfig.LoggingRules.Add(new LoggingRule("Particular.ServiceControl.Licensing.*", LogLevel.Info, consoleTarget));
 
             // Defaults
-            nlogConfig.LoggingRules.Add(new LoggingRule("*", loggingSettings.LoggingLevel, fileTarget));
-            nlogConfig.LoggingRules.Add(new LoggingRule("*", loggingSettings.LoggingLevel < LogLevel.Info ? loggingSettings.LoggingLevel : LogLevel.Info, consoleTarget));
-
-            if (!loggingSettings.LogToConsole)
-            {
-                foreach (var rule in nlogConfig.LoggingRules.Where(p => p.Targets.Contains(consoleTarget)).ToList())
-                {
-                    nlogConfig.LoggingRules.Remove(rule);
-                }
-            }
+            nlogConfig.LoggingRules.Add(new LoggingRule("*", loggingSettings.LogLevel, fileTarget));
+            nlogConfig.LoggingRules.Add(new LoggingRule("*", loggingSettings.LogLevel < LogLevel.Info ? loggingSettings.LogLevel : LogLevel.Info, consoleTarget));
 
             NLog.LogManager.Configuration = nlogConfig;
 
             LogManager.UseFactory(new ExtensionsLoggerFactory(new NLogLoggerFactory()));
 
             var logger = LogManager.GetLogger("LoggingConfiguration");
-            var logEventInfo = new LogEventInfo
-            {
-                TimeStamp = DateTime.UtcNow
-            };
-            logger.InfoFormat("Logging to {0} with LogLevel '{1}'", fileTarget.FileName.Render(logEventInfo), loggingSettings.LoggingLevel.Name);
+            var logEventInfo = new LogEventInfo { TimeStamp = DateTime.UtcNow };
+            logger.InfoFormat("Logging to {0} with LogLevel '{1}'", fileTarget.FileName.Render(logEventInfo), loggingSettings.LogLevel.Name);
         }
+
+        const long megaByte = 1024 * 1024;
```

Comment:
> Use PascalCase for constant names, both fields and local constants.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

That would be my preference but I realize we have no enforcement so it is just my opinion